src/api/api.ts

```diff
@@ -152,3 +154,45 @@ function dataURItoFile(dataURI: string) {
   const blob = new Blob([buffer], { type: mimeString });
   return blob;
 }
+
+//Program Enrollment
+export function getPatientEnrolledPrograms(patientUuid: string) {
+  return openmrsFetch(
+    `${BASE_WS_API_URL}programenrollment?patient=${patientUuid}&v=custom:(uuid,display,program,dateEnrolled,dateCompleted,location:(uuid,display))`,
+  ).then(({ data }) => {
+    if (data) {
+      return data;
+    }
+    return null;
+  });
+}
+
+export function createProgramEnrollment(payload, abortController) {
```

Comment:
Let's define an interface for payload and add type checking to it

Reply:
Yah, makes sense